packages/react-adminer/src/components/Edit.tsx

```diff
@@ -255,7 +268,7 @@ export const Edit: React.FC<Props> = ({ entityConfig, entityName, id }) => {
 									f.nullable,
 									<DatePicker
 										disabled={isDisabled}
-										value={state[f.name]}
+										value={setValidDateValue(f, f.editable)}
```

Comment:
Opravdu to je `setter`? Nemá to být `getter`? Setuješ (nastavuješ)? Nebo getuješ (čteš?).

Tímhle své kolegy budeš hodně pléct :-)a taky to je hnusný